app/bundles/LeadBundle/Assets/js/lead.js

```diff
@@ -282,7 +282,13 @@ Mautic.leadlistOnLoad = function(container) {
         Mautic.activateSearchAutocomplete('list-search', 'lead.list');
     }
 
-    if (mQuery('#leadlist_filters').length) {
+    var prefix = 'leadlist';
```

Comment:
Something in this code is causing both segment filters and DWC filters to throw JS errors around reordering filters. ```Uncaught ReferenceError: el is not defined
    at Object.Mautic.reorderSegmentFilters (app/bundles/LeadBundle/Assets/js/lead.js:359:25)```

Reply:
I fixed the issue: There was an undefined variable `em` that was causing the trouble